static/css/legacy.less

```diff
@@ -1692,6 +1698,7 @@ ul#siteSearch {
   // * openlibrary/templates/books/show.html
   // * openlibrary/templates/books/works-show.html
   span.resultTitle .bookauthor,
+  div.resultTitle .bookauthor,
```

Comment:
I think it would be fine to just do `.resultTitle .bookauthor` here. No need for the span or div qualifier.